### ISSUE TITLE
[Enhancement]get available server list

### ISSUE THREAD

**Issue by davisbra** [ProtonVPN/linux-cli#260]
a command for getting all available server lists (as a string not interactive)
for example :
`sudo protonvpn serverlist`
return :
JP-FREE#1
JP-FREE#2
JP#1
...

**Comment by larobitrumpet** [comment]
I like your idea, @davisbra. I have made a branch with this feature: [list-servers](https://github.com/larobitrumpet/linux-cli/tree/list-servers). There might be a better way of doing it, but it works. Let me know if you want me to make a pull request.

**Comment by 7astro7** [comment]
I like your idea too @davisbra, a command for seeing all available servers seems worthwhile. A command or option to see the set of countries that have available servers is another feature I've felt is missing. A patch along the lines of @larobitrumpet's changes is close to what I've envisioned.

**Comment by nrvale0** [comment]
@larobitrumpet A quick glance suggests this was never submitted as a PR.?.

+1 to the idea, btw.

**Comment by larobitrumpet** [comment]
@nrvale0 That is correct, I never submitted a PR. In #271 I was told "no more commits will be done on this repo".

**Comment by nrvale0** [comment]
Thanks for following up here.
